ssts/segmentation/example.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import segmentation_gmm as seg_gmm
+from methods import main_methods as mm # NOTE SSTS directory must be in PYTHONPATh
```

Comment:
Let go of the shift key too early: `PYTHONPATH`

Reply:
Lol. I fixed it anyway